ngi_pipeline/engines/piper_ngi/database.py

```diff
@@ -63,6 +63,8 @@ def create_database_populate_schema(location):
     if not os.path.exists( os.path.dirname(location) ):
         try:
             os.makedirs( os.path.dirname(location)) 
+        except OperationalError:
```

Comment:
are we sure we want to continue in case we are not able to create the directory?